packages/devtools_app/lib/src/code_size/code_size_screen.dart

```diff
@@ -0,0 +1,87 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+import '../auto_dispose_mixin.dart';
+import '../charts/treemap.dart';
+import '../common_widgets.dart';
+import '../globals.dart';
+import '../octicons.dart';
+import '../screen.dart';
+import 'code_size_controller.dart';
+
+class CodeSizeScreen extends Screen {
+  const CodeSizeScreen() : super(id, title: 'Code Size', icon: Octicons.rss);
+
+  static const id = 'codeSize';
+
+  @override
+  String get docPageId => id;
+
+  @override
+  Widget build(BuildContext context) {
+    final connected = serviceManager?.connectedApp;
```

Comment:
does whether the app is connected really matter for the code size screen?

Reply:
even if we later add some functionality that leverages the running app (e.g. use coverage info) we'll still want people to be able to use this tool without needed a connected app.

Reply:
I see. I removed the condition